.gitignore

```diff
@@ -2,6 +2,7 @@ node_modules/
 .vscode/
 .env*
 .env.local
+!/playwright/.env*
```

Comment:
Once this PR is merged and the `.env.playwright` is being tracked on the main branch, we can then remove this line